Martium.TravelInfo/Forms/TravelInfoForm.cs

```diff
@@ -57,17 +61,186 @@ private void InitializeMap()
 
         private void LoadTravelInfoSettings()
         {
-            TravelInfoSettingsModel travelInfoSettingsModel = _travelInfoRepository.GetExistingInfo();
+            _travelInfoSettingsModel = _travelInfoRepository.GetExistingInfo();
 
-            DepartureCountryTextBox.Text = travelInfoSettingsModel.DepartureCountry;
-            DepartueAddressTextBox.Text = travelInfoSettingsModel.DepartureAddress;
-            PricePerKm.Text = travelInfoSettingsModel.PricePerKm.ToString(CultureInfo.InvariantCulture);
-            AdditionalDistanceInKm.Text = travelInfoSettingsModel.AdditionalDistanceInKm.ToString(CultureInfo.InvariantCulture);
+            DepartureCountryTextBox.Text = _travelInfoSettingsModel.DepartureCountry;
+            DepartureAddressTextBox.Text = _travelInfoSettingsModel.DepartureAddress;
+            PricePerKm.Text = _travelInfoSettingsModel.PricePerKm.ToString(CultureInfo.InvariantCulture);
+            AdditionalDistanceInKmTextBox.Text =
+                _travelInfoSettingsModel.AdditionalDistanceInKm.ToString(CultureInfo.InvariantCulture);
         }
 
+        private void DepartureCountryTextBox_TextChanged(object sender, System.EventArgs e)
+        {
+            if (DepartureCountryTextBox.Text == "LTU")
```

Comment:
converting LTU -> Lietuva should be done in `TravelInfoForm_Load` event single time `_TextChanged` event should be used only for validation/buttons enabling (which is not needed for this text box)